examples/README.md

```diff
@@ -0,0 +1,41 @@
+# Tinyman JS SDK Examples
+
+This is a collection of node scripts that uses tinyman-js-sdk v2 operations.
+
+⚠️ Please be aware that the scripts are not production-ready, and are only for testing purposes. If you plan to use the scripts in production, please make sure to test and optimize them for your use case.
+
+### File/Folder Structure
+
+- `operation` folder includes scripts that perform operations on the pool
+- `util` folder includes helper functions or constant variables that are used in the test scripts
+- `index.ts` includes a `main` function that runs all of the operations in order;
+  1. Create an account and two assets (for testing purposes)
```

Comment:
Should we just use an account address and two testnet assets instead of creating them? I think this folder should just be about demonstrating how to make use of the library and these are little unnecessary. We can simply the steps by just using addresses and ids. For the address, we could do something like `const ACCOUNT_ADDRESS = "YOUR_ACCOUNT_ADDRESS";` like we do for tokens below. Then, the step one would be using `getAccountInformation` which is more proper and realistic in terms of real-life usage of the library.

Reply:
>  For the address, we could do something like const ACCOUNT_ADDRESS = "YOUR_ACCOUNT_ADDRESS"; like we do for tokens below. Then, the step one would be using getAccountInformation which is more proper and realistic in terms of real-life usage of the library.


`getAccount()` returns data in `Account`  type, which is:
```tsx
interface Account {
    /**
     * Algorand address
     */
    addr: string;
    /**
     * Secret key belonging to the Algorand address
     */
    sk: Uint8Array;
}

```

To be able to make signing possible on node, we need `sk`, the secret key.
I think it's not very easy to get sk of your already existing account (for example I couldn't find a way to do it 😄 )
So I believe we can keep the account and asset creation steps, otherwise examples will be really hard to run.

---

> Should we just use an account address and two testnet assets instead of creating them? 

I don't think this would be useful since we aim to show the whole flow from start to end;
1. create pool
2. add initial liquidity
3. add subsequent liquidity
....

1 and 2 requires new assets

Reply:
We can generate secret key using a mnemonic but I get your point 😄 
https://algorand.github.io/js-algorand-sdk/functions/mnemonicToSecretKey.html